assets/components/atoms/social-icon/social-icon-round-discrete-link.twig

```diff
@@ -0,0 +1,9 @@
+<a href="#" class="social-icon social-icon-facebook social-icon-discrete social-icon-round">
```

Comment:
In this case and because the structure is so close to any variant, you can add a `class` Twig variable, then use a variant component to create variant 😉 

ex:

```twig
{% include "atoms::social-icon/social-icon-link.twig" with { classes: 'social-icon-discrete social-icon-round' } %}
```

Reply:
Yes, more handy to handle in the styleguide, but that way it gives a better visual on what's available, for end users.

Reply:
No no, the result wont' change. Do something like : 

./assets/components/atoms/social-icon/social-icon-link.twig
```twig
<a href="#" class="social-icon social-icon-facebook {{classes}}">
  {% include 'atoms::icon/icon.twig' with { icon: 'icon-facebook' } %}
</a>
<a href="#" class="social-icon social-icon-twitter {{classes}}">
  {% include 'atoms::icon/icon.twig' with { icon: 'icon-twitter' } %}
</a>
<a href="#" class="social-icon social-icon-instagram {{classes}}">
  {% include 'atoms::icon/icon.twig' with { icon: 'icon-instagram' } %}
</a>
<a href="#" class="social-icon social-icon-mail {{classes}}">
  {% include 'atoms::icon/icon.twig' with { icon: 'icon-mail-plane' } %}
</a>
<a href="#" class="social-icon social-icon-whatsapp {{classes}}">
  {% include 'atoms::icon/icon.twig' with { icon: 'icon-whatsapp' } %}
</a>
<a href="#" class="social-icon social-icon-googleplus {{classes}}">
  {% include 'atoms::icon/icon.twig' with { icon: 'icon-googleplus' } %}
</a> 
```

./assets/components/atoms/social-icon/social-icon-round-discrete-link.twig
```twig
{% include "atoms::social-icon/social-icon-link.twig" with {
  classes: 'social-icon-discrete social-icon-round'
} %}
```

It will be soooo much easier to maintain 😉

Reply:
I get what you mean now ^_^ agreed!